DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -726,7 +726,8 @@ internal async Task OnChannelPinsUpdateAsync(ulong? guildId, ulong channelId, Da
                 {
                     Id = channelId,
                     Discord = this,
-                    Type = ChannelType.Private
+                    Type = ChannelType.Private,
+                    Recipients = new DiscordUser[] {}
```

Comment:
Wouldn't `Array.Empty<DiscordUser>()` be more appropriate here other places so we won't allocate array needlessly?

Reply:
Correcc